.github/workflows/publish_package.yml

```diff
@@ -0,0 +1,18 @@
+name: Node.js Package
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
```

Comment:
I don't understand this comment

Reply:
Ah, I took this from the boilerplate, see: https://docs.github.com/en/actions/guides/publishing-nodejs-packages#publishing-packages-to-the-npm-registry

Reply:
I see. So the step does create an `.npmrc` file... I wonder how it does that. We already have an `.npmrc` file in this repo, hopefully that won't interfere with this step.